vignettes/SubDailyTimeSteps.Rnw

```diff
@@ -141,11 +146,14 @@ Load estimation can easily be done using unit-value data, rather than the user a
 
 <<echo=TRUE>>=
 # Get the data for May, 2013
-BrushyQ <- readNWISuv(Brushy, "00060", startDate="2013-05-01", 
+BrushyQ <- readNWISuv(Brushy, "00060", startDate="2013-05-01",
   endDate="2013-05-31", tz="America/New_York")
 # Rename the Flow column, must be done manually as renameNWISColumns
 # appends _Inst
-names(BrushyQ)[5] <- "Flow"
+flowColumn <- seq(4, 6, 1)[
+                which.min(lapply(BrushyQ[,4:6],
+                  function(x) sum(is.na(as.numeric(x)))))]
+names(BrushyQ)[flowColumn] <- "Flow"
```

Comment:
@jsta Yes, the columns will come in as X_<param code>_<stat code>.  There may be some edge case sites where there is some extra stuff as well, but for a vignette if you are just pulling particular sites that should be a reasonable assumption.  `renameNWIScolumns` would be an option too.